public/app/plugins/panel/table/transformers.ts

```diff
@@ -254,7 +285,8 @@ function transformDataToTable(data: any, panel: any) {
     throw { message: 'Transformer ' + panel.transform + ' not found' };
   }
 
-  transformer.transform(data, panel, model);
+  const reducedData = transformer.dataFormatReducer ? transformer.dataFormatReducer(data) : data;
```

Comment:
does this step need to be done outside the transform? maybe cleaner if this is done inside the transform function instead.

Reply:
Ok I can see why that could potentially become a problem if transform is called without the filtering. Will change!